template.tpl

```diff
@@ -810,185 +1115,22 @@ ___TEMPLATE_PARAMETERS___
-                            "value": "all",
-                            "displayValue": "All history changes"
-                          },
-                          {
-                            "value": "path",
-                            "displayValue": "Only when page path changes"
-                          }
-                        ],
-                        "simpleValueType": true,
-                        "help": "Provides advanced control for single page application on when page views are tracked. \u003ca href\u003d\"#advanced-configuration-for-tracking-page-views\"\u003eRead more\u003c/a\u003e.",
-                        "defaultValue": "all"
-                      }
-                    ],
-                    "enablingConditions": [
-                      {
-                        "paramName": "detPageView",
-                        "paramValue": true,
-                        "type": "EQUALS"
-                      }
-                    ]
-                  }
-                ],
-                "help": "Check this box to enable page views tracking. \u003ca href\u003d\"https://www.docs.developers.amplitude.com/data/sdks/browser-2/#tracking-page-views\"\u003eRead more\u003c/a\u003e.",
-                "defaultValue": true
-              },
-              {
-                "type": "CHECKBOX",
-                "name": "detSession",
-                "checkboxText": "Enable session tracking",
-                "simpleValueType": true,
-                "help": "Check this box to enable session tracking. \u003ca href\u003d\"https://www.docs.developers.amplitude.com/data/sdks/browser-2/#tracking-sessions\"\u003eRead more\u003c/a\u003e.",
-                "defaultValue": true
-              },
-              {
-                "type": "CHECKBOX",
-                "name": "detFormInteraction",
-                "checkboxText": "Enable form interaction tracking",
-                "simpleValueType": true,
-                "help": "Check this box to enable form interaction tracking. \u003ca href\u003d\"https://www.docs.developers.amplitude.com/data/sdks/browser-2/#tracking-form-interactions\"\u003eRead more\u003c/a\u003e.",
-                "defaultValue": true
-              },
-              {
-                "type": "CHECKBOX",
-                "name": "detFileDownload",
-                "checkboxText": "Enable file download tracking",
-                "simpleValueType": true,
-                "help": "Check this box to enable file download tracking. \u003ca href\u003d\"https://www.docs.developers.amplitude.com/data/sdks/browser-2/#tracking-file-downloads\"\u003eRead more\u003c/a\u003e.",
-                "defaultValue": true
-              }
-            ],
-            "enablingConditions": [
-              {
-                "paramName": "defaultEventTracking",
-                "paramValue": true,
-                "type": "EQUALS"
-              }
-            ]
-          }
-        ]
+        "valueHint": "Default (blank)",
```

Comment:
i'm thinking we can replace this it "Main instance" or something to add a little bit of context. "Default (blank)" isn't very clear and I feel like it would just adds confusion

Reply:
How about "Main (Default)"?

Reply:
yup thas ok